src/hydrate/platform/index.ts

```diff
@@ -155,7 +155,7 @@ export const Build: d.UserBuildConditionals = {
 export const styles: d.StyleMap = new Map();
 export const modeResolutionChain: d.ResolutionHandler[] = [];
 
-export { hAsync as h } from './h-async';
+export { hAsync as h1 } from './h-async';
```

Comment:
So...this is interesting. This might actually be a blocker/breaking change Prettier causes

Reply:
Maybe this somehow happened while I was looking at the design doc for `Host` attributes yesterday (I unintentionally added the '1' to the export and had unstaged changes). I reverted back to https://github.com/ionic-team/stencil/pull/4586/commits/b865a0744db698d4591ca9948f41ef295039cd3b and reran the formatter, and _didn't_ see `h` get transformed to `h1`. 

Regardless, the test suite caught this, which is comforting 🙂 I've rerun the formatter and everything looks OK now